parastell/nwl_utils.py

```diff
@@ -186,6 +186,11 @@ def flux_coords(plas_eq, wall_s, coords, num_threads):
             for theta_coord_chunk in theta_coord_chunks
             for theta_coord in theta_coord_chunk
         ]
+    # Ensures theta_coords are all positive (add 360 degrees where needed)
+    theta_coords = [
+        (theta if theta >= 0 else theta + 2 * np.pi) for theta in theta_coords
+    ]
```

Comment:
```suggestion
    theta_coords = (theta_coords + 2 * np.pi) % 2*np.pi
```
Avoids the use of conditionals

Reply:
You should be able to directly commit this suggestion

Reply:
That's pretty slick!